script.trakt/resources/lib/kodiUtilities.py

```diff
@@ -0,0 +1,356 @@
+# -*- coding: utf-8 -*-
+#
+
+import xbmc
+import xbmcgui
+import xbmcaddon
+import json
+import re
+import sys
+import logging
+from resources.lib import utilities
+
+
+# read settings
+__addon__ = xbmcaddon.Addon('script.trakt')
+
+logger = logging.getLogger(__name__)
+
+REGEX_URL = '(^https?://)(.+)'
+
+def notification(header: str, message: str, time=5000, icon=__addon__.getAddonInfo('icon')):
+    xbmc.executebuiltin("XBMC.Notification(%s,%s,%i,%s)" % (header, message, time, icon))
+
+def showSettings():
+    __addon__.openSettings()
+
+def getSetting(setting):
+    return __addon__.getSetting(setting).strip()
+
+def setSetting(setting, value):
+    __addon__.setSetting(setting, str(value))
+
+def getSettingAsBool(setting):
```

Comment:
should not be needed in newer api versions

Reply:
Can I change that in the next version?

Reply:
sure